### DISCUSSION TITLE
Add ISO 8601 as date format

### DISCUSSION THREAD

**Discussion by JulesRoland** [aleksey-hoffman/sigma-file-manager#223] [Ideas]
Please add the option to format all date and time following ISO 8601 standard, regardless of the regional setting

Speaking from my own experience here, I was born and currently live in Argentina so my regional setting should be Spanish Argentina, but I like my software to display in english, and I like my dates to be displayed following the ISO 8601 standard, currently I've set my regional setting to English (Canada) so I got it working, but it might be confusing for others how to set this option

**Comment by aleksey-hoffman** [comment]
Hi!

Can you please clarify the issue? As you said, you can change the date format in settings to the format you need, so I don't really understand what I could add to improve it.

Are you talking about some additional setting or list of formats? How would it look like?

**Reply by koppor** [reply]
XKCD puts it as follows:

![XKCD-1179](https://www.explainxkcd.com/wiki/images/4/48/iso_8601.png)

Explanation: https://www.explainxkcd.com/wiki/index.php/1179:_ISO_8601

**Comment by JulesRoland** [comment]
Sorry it's a non issue really, but whenever I read regional settings I think about UTC and timezones, that may be a little unclear if somebody wants to use a date format that is not the default for it's country or language

Maybe add an entry called ISO-8601 to the Date / time regional format combo box

Being an international standard, ISO-8601 is not linked to a language/country

**Reply by aleksey-hoffman** [reply]
I see, thanks for the suggestion. I will get back to it if more people request it
At the moment, working on some high priority issues, for the next update
